src/test/java/uk/gov/companieshouse/overseasentitiesapi/service/PublicDataRetrievalServiceTest.java

```diff
@@ -75,40 +88,94 @@ void testInitialisePublicDataIsSuccessful() throws IOException, URIValidationExc
         var companyProfileApi = new CompanyProfileApi();
         var officersApi = new OfficersApi();
         var companyOfficerApi = new CompanyOfficerApi();
+        var pscsApi = new PscsApi();
+        var companyPSCsApi = new PscApi();
```

Comment:
why two vars that are the same thing?

Reply:
I can mock it or change the name but that would affect the consistency with the officers ones, is that fine or is it better to leave it as it is?

Reply:
It's minor leave as is.